packages/default-field-editors/src/FieldWrapper.tsx

```diff
@@ -8,24 +8,26 @@ import { styles } from './FieldWrapper.styles';
 type FieldWrapperProps = {
   name: string;
   sdk: FieldExtensionSDK;
-  getEntryURL: (entry: Entry) => string;
+  getEntryURL?: (entry: Entry) => string;
   className?: string;
   showFocusBar?: boolean;
   children: React.ReactNode;
   renderHeading?: (name: string) => JSX.Element | null;
   renderHelpText?: (helpText: string) => JSX.Element | null;
 };
 
+const defaultGetEntryUrl = (entry: Entry) => `/${entry.sys.id}`;
+
 export const FieldWrapper: React.FC<FieldWrapperProps> = function (props: FieldWrapperProps) {
   const {
     name,
     sdk,
-    getEntryURL,
     className,
     children,
     renderHeading,
     renderHelpText,
     showFocusBar = true,
+    getEntryURL = defaultGetEntryUrl,
```

Comment:
Nit: I still don't understand why/when I should use `getEntryUrl` when using an editor.

Reply:
`getEntryUrl` is required for the "this value is non-unique" validation error, where we fetch a list of duplicate entries and show them in a message together with links to them.

In theory `getEntryUrl` should be a method of SDK, so users won't need to specify one unless they want a custom experience. So I'll add a comment to this prop and also will specify a more proper `defaultGetEntryUrl`.